README.md

```diff
@@ -58,7 +138,7 @@ Python 2.7.10 (default, Aug 13 2015, 12:27:27)
 >>> m.objects.all()                                     
 ```
 
-Note that this is an ongoing issue for which we're planning a more comprehensive fix in [this issue](https://github.com/deis/workflow-e2e/issues/12)).
+Note that this is an ongoing issue for which we're planning [a more comprehensive fix](https://github.com/deis/workflow-e2e/issues/12).
```

Comment:
Note that #12 is closed, but there are a few other related issues, PRs and commits:

- https://github.com/deis/workflow-e2e/issues/142
- https://github.com/deis/workflow-e2e/pull/146 - closed #142, but was reverted because it caused tests to hang on cleanup
- https://github.com/jchauncey/workflow-e2e/commit/116bf930b7d04d82461f99d2b7294a8525e26ec5 reverted #146
- https://github.com/deis/k8s-claimer/issues/39 - for clusters that are leased by our cluster leasing system, all namespaces (including `deis`) will be cleaned up, which will also clean up all apps, since they are stored in ephemeral storage inside the `deis` namespace

Anyway, nothing actionable in this PR. This comment is purely informational. I've created https://github.com/deis/workflow-e2e/issues/162 to track this work.